Terminal.Gui/Views/DateField.cs

```diff
@@ -47,9 +48,11 @@ public class DateField : TextField {
 		/// <param name="y">The y coordinate.</param>
 		/// <param name="date">Initial date contents.</param>
 		/// <param name="isShort">If true, shows only two digits for the year.</param>
-		public DateField (int x, int y, DateTime date, bool isShort = false) : base (x, y, isShort ? 10 : 12, "")
+		/// <param name="isJalaali">If true, parse will convert jalaali input fo georgian date</param>
+		public DateField (int x, int y, DateTime date, bool isShort = false, bool isJalaali = false) : base (x, y, isShort ? 10 : 12, "")
```

Comment:
A few unit tests that prove the various constructors work as they should would be good.